qgispluginci/cli.py

```diff
@@ -247,4 +248,8 @@ def cli():
         t.update_strings()
         t.push()
 
+    else
+        logger.error(f"Unsupported command {args.command}")
+        exit_val = 1
```

Comment:
I'm not sure about this one. Maybe `argparse` actually fails by itself if an invalid subparse command was used.